composer.json

```diff
@@ -20,6 +20,7 @@
   },
   "require": {
     "php": ">=5.3.3",
-    "apache/log4php": "2.3.0"
+    "apache/log4php": "2.3.0",
```

Comment:
I don't think you need to add all PSR3 implementors as "suggested" just to be fair. If you wan't you could require the "psr/log" package that holds the interface and all implementations "provide" this interface: https://packagist.org/packages/psr/log

But as you say, this could be optional for v2.0.0. If a logger is provided it is used, if not you don't nothing is logged.
Monolog could be "suggesed", as this is a very competent logger.